content/patterns/contents.lr

```diff
@@ -260,3 +260,36 @@ To support this:
 
 The `foreignKey` MAY have a property `datapackage`. This property is a string being a url pointing to a Data Package or is the name of a datapackage.
```

Comment:
Should "data package" be "data package" or "Data Package"?

Reply:
I suppose it should be `package` because other property is a `resource`. Also I think this sentence should include Data Package Identifier in the mix somehow.